<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhallLivePlugInUnitDlg</class>
 <widget class="QDialog" name="VhallLivePlugInUnitDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="left_top" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="right_top" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="left" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>898</width>
       <height>602</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>45</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(43,44,46)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_tool">
           <property name="styleSheet">
            <string notr="true">color:rgb(150,150,150);
font: 12pt &quot;微软雅黑&quot;;</string>
           </property>
           <property name="text">
            <string>直播插件</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>840</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <layout class="QHBoxLayout" name="layout_tool"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_10" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(43,44,46)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_web"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="right" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="left_down" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>2</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="down" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QWidget" name="right_down" native="true">
     <property name="minimumSize">
      <size>
       <width>2</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>2</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(60,60,60);</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
